crates/atuin/src/command/client/search/interactive.rs

```diff
@@ -1280,8 +1428,8 @@ mod tests {
         let no_preview = State::calc_preview_height(
             &settings_preview_auto,
             &results,
-            0 as usize,
-            0 as usize,
+            0_usize,
```

Comment:
Just noticing some of these came through from linting but are not related to the current PR - might be a rustfmt version/config difference, so shall I just revert in this PR?